src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<blockquote>This is a very basic and inefficient way of optimizing the aircraft<code>,</code> as we are bound to explore the whole design space, even if the trend seems to suggest that a minimum won't be found here.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Changed in dfe233f7